_includes/header.html

```diff
@@ -11,15 +11,15 @@
 		  </span>
 		</label>
 		<div>
-			<a href="/Sandbox"><img class="header-logo" src="/img/dlf-aig.png"/></a>
```

Comment:
You accidentally removed the quote. Need to add quote after `"/` for the logo to show.